.hass/config/ui-lovelace.yaml

```diff
@@ -75,6 +75,9 @@ kiosk_mode:
     - entity:
         input_boolean.kiosk_block_mouse: 'on'
       block_mouse: true
+    - entity:
+        input_boolean:kiosk_block_context_menu: 'on'
```

Comment:
This will throw an error because it should be input_boolean.kiosk_block_context_menu (with a dot).